anitya/lib/backends/cran.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+
+"""
+ (c) 2018 - Copyright Elliott Sales de Andrade
+
+ Authors:
+   Elliott Sales de Andrade <quantum.analyst@gmail.com>
+
+"""
+
+
+from anitya.lib.backends import BaseBackend
+from anitya.lib.exceptions import AnityaPluginException
+
+
+def _normalize_version(version):
+    return version.replace('-', '.')
+
+
+class CranBackend(BaseBackend):
+    ''' The custom class for projects hosted on CRAN.
+
+    This backend allows to specify a version_url and a regex that will
+    be used to retrieve the version information.
+    '''
+
+    name = 'CRAN (R)'
+    examples = [
+        'https://cran.r-project.org/web/packages/tidyverse/',
+        'https://cran.r-project.org/web/packages/devtools/',
+    ]
+
+    @classmethod
+    def get_version(cls, project):
+        ''' Method called to retrieve the latest version of the projects
+        provided, project that relies on the backend of this plugin.
+
+        :arg Project project: a :class:`anitya.db.models.Project` object whose backend
+            corresponds to the current plugin.
+        :return: the latest version found upstream
+        :return type: str
+        :raise AnityaPluginException: a
+            :class:`anitya.lib.exceptions.AnityaPluginException` exception
+            when the version cannot be retrieved correctly
+
+        '''
+        url = 'https://crandb.r-pkg.org/{name}'.format(name=project.name)
+
+        try:
+            response = cls.call_url(url)
+        except Exception:  # pragma: no cover
```

Comment:
Since this is always an https url you can just except ``requests.RequestException`` here. It'd also be good to include the exception's message in the AnityaPluginException message. Dropping the ``# pragma: no cover`` and adding a test would be good, too.